apps/meteor/app/otr/client/rocketchat.otr.room.js

```diff
@@ -72,15 +80,14 @@ OTR.Room = class {
 	end() {
 		this.isFirstOTR = true;
 		this.reset();
+		this.setState(OtrRoomState.NOT_STARTED);
 		Notifications.notifyUser(this.peerId, 'otr', 'end', {
 			roomId: this.roomId,
 			userId: this.userId,
 		});
 	}
 
 	reset() {
-		this.establishing.set(false);
```

Comment:
Unncessary?

Reply:
Before these new OTR States, reset function reseted the whole OTR room. 

But now we have to maintain different OTR states after reset and we want to know why was it reseted to show in contextual bar, OTR room can be rested and can be in many states like declined, timeout etc. So instead of updating state from inside reset function, I do it whenever reset is called with the state in which OTR is in.